src/main/java/com/elixr/poc/rest/controller/UserRetrievalController.java

```diff
@@ -18,7 +18,6 @@
  */
 
 @RestController
-@Validated
 @RequestMapping("/application")
```

Comment:
Do you need this /application path? I don't think it matches the requirements

Reply:
As it is not in requirement we removed that path